com.github.rigellute.spotify-tui.yaml

```diff
@@ -0,0 +1,28 @@
+command: spt
+id: com.github.rigellute.spotify-tui
+separate-locales: false
+finish-args:
```

Comment:
This has no audio access...?

Reply:
Because it can't play audio. It basically let you switch musing from terminal

Reply:
This is just a cli frontend for spotify, you still need some daemon/program to play the music, like Spotify or spotifyd.